carbon-design-system/carbon
# fix: definition tooltip breaks with large text

Closes #17848 

Definition Tooltip breaks with large text

#### Changelog

1. Update default `align` value to `bottom` instead of `bottom-start`
2. Added autoAlign prop

#### Testing / Reviewing

1. Go to Storybook > Definition Tooltip > Default
2. Hover on text, it should open the tooltip.
